engine/src/main/scala/cromwell/webservice/CromwellApiService.scala

```diff
@@ -184,4 +185,16 @@ trait CromwellApiService extends HttpService with PerRequestCreator {
         }
       }
     }
-}
\ No newline at end of file
+
+  def backendRoute =
+    path("workflows" / Segment / "backends") { version =>
+      get {
+        val backends = BackendConfiguration.AllBackendEntries map { _.name }
+        complete {
+          JsObject(Map("backends" -> backends.toJson))
+        }
+      }
+    }
+
+  val backends = BackendConfiguration.AllBackendEntries map { _.name } toJson
```

Comment:
I considered both sorting the list alphabetically as well as including what the default was. I didn't need this for my purposes so laziness took over, but throwing it out there for reviewers